[DOCS] Creating a Checkpoint from an In-Memory Dataframe (#7701)
great-expectations/great_expectations: docs/docusaurus/docs/guides/validation/checkpoints/how_to_pass_an_in_memory_dataframe_to_a_checkpoint.md

### 1. Set up Great Expectations
#### Import the required libraries and load your DataContext

```python name="tests/integration/docusaurus/validation/checkpoints/how_to_pass_an_in_memory_dataframe_to_a_checkpoint.py setup"
```

### 2. Read a DataFrame and create a Checkpoint

> The following example uses the `read_*` method on the PandasDatasource to directly return a <TechnicalTag tag="validator" text="Validator" />. Validators can be used to interactively build an Expectation Suite, as described in [How to create Expectations interactively in Python](/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data).

Reviewer: ```suggestion
The following example uses the `read_*` method on the PandasDatasource to directly return a <TechnicalTag tag="validator" text="Validator" />. To use Validators to interactively build an Expectation Suite, see [How to create Expectations interactively in Python](/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data).
```

The Validator can be passed directly to a SimpleCheckpoint

```python name="tests/integration/docusaurus/validation/checkpoints/how_to_pass_an_in_memory_dataframe_to_a_checkpoint.py read_dataframe"
```

> Alternatively, you can use the `add_*` methods to add the asset, then retrieve a <TechnicalTag tag="batch_request" text="Batch Request" />. This method is more consistent with how other Data Assets work, and can more easily integrate in-memory data with other Batch Request workflows and configuration.

Reviewer: ```suggestion
Alternatively, you can use `add_*` methods to add the asset and then retrieve a <TechnicalTag tag="batch_request" text="Batch Request" />. This method is consistent with how other Data Assets work, and can integrate in-memory data with other Batch Request workflows and configurations.
```

```python name="tests/integration/docusaurus/validation/checkpoints/how_to_pass_an_in_memory_dataframe_to_a_checkpoint.py add_dataframe"
```

> In both examples, `batch_metadata` is an optional parameter that can associate meta-data with the batch (or DataFrame). This can be useful in distinguishing Validation results when working with DataFrames.

Reviewer: ```suggestion
In both examples, `batch_metadata` is an optional parameter that can associate meta-data with the batch or DataFrame. When you work with DataFrames, this can help you distinguish Validation results.
```
